modules/gitops-monitoring-argo-cd-health-using-prometheus-metrics.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * cicd/gitops/monitoring-argo-cd-instances.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-monitoring-argo-cd-health-using-promethous-metrics_{context}"]
+= Monitoring Argo CD health using Prometheus metrics
+
+You can monitor the health status of an Argo CD application by running Prometheus metrics queries against it.
+
+.Procedure
+
+. In the *Developer* perspective of the web console, select the namespace where your Argo CD application is installed, and navigate to *Observe* -> *Metrics*.
+. From the *Select query* drop-down list, select *Custom query*.
+. To check the health status of your Argo CD application, enter the Prometheus Query Language (PromQL) query similar to the following example in the *Expression* field:
++
+.Example
+[source, query]
```

Comment:
I do not see `[source,query]` anywhere else in the docs. I think we would use `[source,terminal]`. Again, if `source,query` is correct for your team, please disregard this comment.